ctms/bin/prepare_csvs.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+"""Given a dump of csvs from bigquery, prepare it for being imported
+into postgres with COPY."""
```

Comment:
Nit: I was surprised that the linter was OK with this style. I'd expect:
```suggestion
"""
Given a dump of csvs from bigquery, prepare it for being imported
into postgres with COPY.
"""
```